pkg/kubelet/kubelet.go

```diff
@@ -2604,6 +2604,7 @@ func (kl *Kubelet) HandlePodAdditions(pods []*v1.Pod) {
 				// Check if we can admit the pod; if not, reject it.
 				if ok, reason, message := kl.canAdmitPod(activePods, pod); !ok {
 					kl.rejectPod(pod, reason, message)
+					metrics.AdmissionRejectionsTotal.WithLabelValues(reason).Inc()
```

Comment:
to avoid proliferation of labels we should ensure (or check) the "reason" doesn't contain any pod-related information like pod name or so. I'm not sure we can guarantee this.

Reply:
I manually verified, but we could put an explicit allowlist here to guarantee it.

Reply:
As the function's design and description, the reason is a brief single-word. 
```
// The function returns a boolean value indicating whether the pod
// can be admitted, a brief single-word reason and a message explaining why
// the pod cannot be admitted.
```